internal/commands/pull/command.go

```diff
@@ -258,17 +258,22 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
-	successfulTemplateWrites := make([]string, 0, len(clientTemplates))
+	successfulTemplateWrites := make([]interface{}, 0, len(clientTemplates))
 	for _, ct := range clientTemplates {
 		if err := local.WriteZip(filepath.Join(pathFrontend, ct.id), ct.zipPkg); err != nil {
 			return fmt.Errorf("unable to save template '%s' to disk: %s", ct.id, err)
 		}
-		successfulTemplateWrites = append(successfulTemplateWrites, ct.id)
+		readmeFile := ""
```

Comment:
[style/meganit/optional] in the theme of "make the zero value meaningful", I tend to prefer declarations like this as `var readmeFile string` instead of assigning `:= ""` to the literal zero value